src/js/model/storage.ts

```diff
@@ -1,30 +1,38 @@
 import { serialize } from 'utils/parser';
 import ApiSettings from 'api/api-settings';
+import type { GenericObject } from 'types/generic.type';
+import type { MediaModel } from 'controller/model';
 
 let storage = {
-    removeItem: function() {}
+    removeItem: function(itemName: string) {
+        // No unused arguments.
```

Comment:
I remember seeing this issue before and this is what we did in plugins/utils.
https://github.com/jwplayer/jwplayer/blob/238ff21f60af85c85943bfcf1e8fc7575fc17ff8/src/js/plugins/utils.ts#L9-L12

Reply:
awesome, glad to know that's an option. Thanks for following up!